auction-server/src/server.rs

```diff
@@ -380,6 +381,7 @@ fn setup_svm(
             (
                 chain_id.clone(),
                 ChainStoreSvm {
+                    recent_blockhash: None.into(),
```

Comment:
We can store the initial block hash by creating a single RPC request when setting up the server. It's better to not let this value be None